exporters/metric/prometheus/prometheus.go

```diff
@@ -130,13 +130,13 @@ func NewExportPipeline(config Config, options ...controller.Option) (*Exporter,
 // InstallNewPipeline instantiates a NewExportPipeline and registers it globally.
 // Typically called as:
 //
-// 	hf, err := prometheus.InstallNewPipeline(prometheus.Config{...})
+// 	exporter, err := prometheus.InstallNewPipeline(prometheus.Config{...})
 //
 // 	if err != nil {
 // 		...
 // 	}
-// 	http.HandleFunc("/metrics", hf)
-// 	defer pipeline.Stop()
+// 	http.HandleFunc("/metrics", exporter)
+// 	defer exporter.Controller().Stop(context.TODO())
```

Comment:
Could we move this to an example test to allow the compiler to help us ensure it stays up-to-date in the future?